detox/src/devices/DeviceRegistry.test.js

```diff
@@ -2,6 +2,33 @@ const fs = require('fs-extra');
 const tempfile = require('tempfile');
 const environment = require('../utils/environment');
 
+const deviceHandle = 'emulator-5554';
+
+const deviceHandleAsObjSanitized = {
```

Comment:
Why "sanitized"? What hacker attack we are preventing?

Reply:
Lol. I will rename across the board. What would be the proper terminology? "clean"? Stripped?!